config/manager/manager.yaml

```diff
@@ -30,8 +30,8 @@ spec:
         - -v 5
         env:
         - name: RELATED_IMAGE_DEX_OPERATOR
-          value: "quay.io/identitatem/dex-operator@sha256:b5f33d7939f765cabfa866b9bc74d1071c633bc308ed27288f7b84999f459a81"
-        image: quay.io/identitatem/idp-mgmt-operator@sha256:e499da1e4a2ddf6c4845bab8c9e21978487132c8b1dae5082dbe6739a5b15caf
+          value: "quay.io/identitatem/dex-operator@sha256:32829b387b9a7c490dcf67461b72597c6fa22c23ea5997bc8a53bcd11e6c043f"
+        image: quay.io/identitatem/idp-mgmt-operator@sha256:bf101fed1de2da84e200091748267faff1ce635bc73d6c541eea562719ba30c5
```

Comment:
assume you only want dex-operator pinned to specific version?

Reply:
ok.